### PR TITLE
Fabo/1911 add sentry version

### LINKED ISSUE #1911: Version not set in Sentry
image

### PR DESCRIPTION
**Pull request by faboweb**
Closes #1911 

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/renderer/vuex/modules/user.js [hunk 1]
```diff
@@ -174,7 +174,7 @@ export default ({}) => {
       if (state.errorCollection) {
         state.externals.Sentry.init({
           dsn: config.sentry_dsn,
-          release: `voyager@${state.externals.config.version}`
+          release: process.env.RELEASE
```
Comment (fedekunze) [inline app/src/renderer/vuex/modules/user.js]: do we want to add the release or the commit ?

**Comment by fedekunze** [comment]
test fix missing

**Comment by faboweb** [comment]
I manually created a version running everything locally. Let's see if this works when merging into develop.

**Comment by sabau** [comment]
In this way we are building twice the same code right?

**Comment by faboweb** [comment]
> In this way we are building twice the same code right?

yeah, that was the simplest solution (webpack plugin). The upload takes quiet some time so I didn't want to run it always (not on each feature, only on develop/master).

**Pull request merged**